custom/succeed/data_sources/patient_task_list.json

```diff
@@ -0,0 +1,142 @@
+{
+    "domains": ["succeed"],
+    "config": {
+        "referenced_doc_type": "CommCareCase",
+        "table_id": "patient_task_list",
+        "display_name": "Patient Tasks List",
+         "configured_filter":  {
+            "type": "boolean_expression",
+            "operator": "eq",
+            "expression": {
+                "type": "property_name",
+                "property_name": "type"
+            },
+            "property_value": "task"
+        },
+        "configured_indicators": [
+            {
+                "column_id": "referenced_id",
+                "type": "expression",
+                "expression": {
```

Comment:
to clarify my comment below, I think this can be replaced with:
```json
{
    "type": "related_doc",
    "related_doc_type": "CommCareCase",
    "doc_id_expression": {
        "type": "succeed_referenced_id"
    },
    "value_expression": {
        "type": "property_name",
        "property_name": "full_name"
    }
}
```